src/pages/administrar-demandas/administrar-demandas.css

```
@@ -0,0 +1,320 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    background-color: #E7ECEF;
+    min-height: 800px
+}
+
+.content {
+    background-color: #E7ECEF;
+    border-radius: 0;
+    display: block;
+}
+
+.content > .headline > .title {
+    color: #274C77;
+    padding: 16px 24px 0 24px;
+    font-size: 40px;
+    text-align: left;
```

acho que o title no meio combinaria melhor com as outras pages, o que acha? 
```suggestion
    text-align: center;
```

---

Apliquei a alteração :)